package.json

```diff
@@ -51,5 +51,8 @@
     "lint": "bin/lint",
     "release": "bin/release",
     "test": "npm run lint && bin/test && npm run doctest"
+  },
+  "mocha": {
+    "ui": "tdd"
```

Comment:
I think this has to be added as a command line argument, here:

https://github.com/sanctuary-js/sanctuary-scripts/blob/545d2feacf5f490723c2fe029dce4eb1192012de/bin/test#L21

Otherwise I don't think it's going to be applied in dependent projects.

Reply:
Thx for the hin.

Reply:
Ah, I forgot about `mocha.opts`, sorry for the misdirection. See https://github.com/sanctuary-js/sanctuary-scripts/pull/41#discussion_r609565971

Reply:
I think I've come full circle. Your original code was correct (prior to https://github.com/sanctuary-js/sanctuary-scripts/pull/41/commits/bf9fb04eb76d3e2e65c5d09788995a3bea60944f). We just need to tell other users of the lib to use the same approach. Currently the README advertises using `mocha.opts`. What do you think @davidchambers ?

Reply:
I agree, Aldwin. I suggest the following:

> Configurable via [variables][1] (`min-branch-coverage`) and the `mocha` field of __package.json__.


[1]: https://github.com/sanctuary-js/sanctuary-scripts#variables